themes/README.rst

```diff
@@ -0,0 +1,146 @@
+#####################
```

Comment:
I may be out of the loop, but is this where the final repo documentation for theming will live? If so, is this a convention I just don't know about? Do people who are trying to theme know to look here?